javascripts/show/show.js

```diff
@@ -167,7 +171,7 @@ class FeaturedContentPopOutWidget {
   }
 
   toggleFeaturedContentPopOut() {
-    this.widget.find('.featured-content-pop-out-aside, .featured-content-pop-out-overlay').toggleClass('open');
+    this.toggleBanner()
```

Comment:
The function `toggleBanner()` doesn't seem to account for the selector `.featured-content-pop-out-overlay` that is being removed here.  Is removing this selector going to cause any issues when it now doesn't get the `open` class applied?

Reply:
Thanks for seeing that!